etc/templates/nginx.conf.system.template

```diff
@@ -2,16 +2,18 @@
         listen       %(HTTPPORT)d;
         server_name  $hostname;
 
+        error_page 502  /500.html;
```

Comment:
This sets the error_page for `/`, but if a prefix has been specified shouldn't the error handler only apply to e.g. `/omero`?